packages/discord.js/src/structures/Embed.js

```diff
@@ -1,11 +1,195 @@
 'use strict';
 
-const { Embed: BuildersEmbed } = require('@discordjs/builders');
-const Transformers = require('../util/Transformers');
+const isEqual = require('fast-deep-equal');
 
-class Embed extends BuildersEmbed {
-  constructor(data) {
-    super(Transformers.toSnakeCase(data));
+class Embed {
+  /**
+   * Creates a new embed object
+   * @param {APIEmbed} data API embed data
+   * @private
+   */
+  constructor(data = {}) {
```

Comment:
```suggestion
  constructor(data) {
```
`{ ...undefined }` in L17 works fine